SDVModTest/ModEntry.cs

```diff
@@ -66,27 +67,25 @@ private void OnReturnedToTitle(object sender, ReturnedToTitleEventArgs e)
         /// <param name="e">The event arguments.</param>
         private void OnSaved(object sender, EventArgs e)
         {
-            if (!String.IsNullOrWhiteSpace(_modDataFileName))
+            if (string.IsNullOrWhiteSpace(_modDataFileName)) return;

```

Comment:
Sorry but I hate multiple returns. I'm willing to accept guard statements like this sometimes and maybe if you'd submitted this as a separate MR I'd be willing to talk it over with you but I'm not ok with it in this huge MR full of things you didn't need to change.